Fabric.Terminology.API/Scripts/Install-Terminology.ps1

```diff
@@ -34,39 +34,39 @@
     [switch] $Quiet
 )
 
-[string[]] $requiredFiles = @($InstallFile, $Dacpac, $PublishProfile, "$PSScriptRoot\Install.config", "$PSScriptRoot\Register.ps1", "$PSScriptRoot\Terminology-Install-Utilities.psm1", "$PSScriptRoot\registration.config")
+# Fail installation on first error
+$ErrorActionPreference = "Stop"
```

Comment:
This is a built in variable in powershell

Reply:
Copy that